### ISSUE TITLE
Move rewriter - sink mapping from solrconfig.xml to rewriter config

### ISSUE THREAD

**Issue by renekrie** [querqy/querqy#387]
Compare ES
```
{
 "class": "querqy.RewriterClass",
"info_logging": { 
   "sinks": ["log4j"]
},
 "config": {
     "rules" : "notebook =>\nSYNONYM: laptop"
 }
}
```

**Comment by renekrie** [comment]
#375

**Comment by renekrie** [comment]
Implemented by #388
